src/main/kotlin/uk/gov/justice/digital/hmpps/managerecallsapi/controller/RecallController.kt

```diff
@@ -334,14 +342,18 @@ data class Pdf(val content: String) {
 }
 
 data class RecallSearchRequest(val nomsNumber: NomsNumber)
-enum class Status {
-  BEING_BOOKED_ON,
-  BOOKED_ON,
-  IN_ASSESSMENT,
-  RECALL_NOTIFICATION_ISSUED,
-  DOSSIER_IN_PROGRESS,
-  DOSSIER_ISSUED,
-  STOPPED
+
+private val ALL_BANDINGS = setOf(THREE, FOUR_PLUS)
+private val FOUR_PLUS_ONLY = setOf(FOUR_PLUS)
+
+enum class Status(val bandings: Set<CaseworkerBand>) {
```

Comment:
I find the term `bandings`, err, a bit 🤯 .  It doesn't tell me anything of what it means TBH.  Even just `visibility` would be preferable?
Perhaps `visibilityBands`?